commands/command_clean.go

```diff
@@ -44,6 +44,10 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 		file.Close()
 	}
 
+	if err != nil {
+		Panic(err, "Error cleaning asset.")
```

Comment:
The same check is [a few lines below this](https://github.com/github/git-lfs/blob/923be66fdd733be438a3fbdd1eb1201ec24bda0c/commands/command_clean.go#L57-L59).  But first, it attempts to close the cleaned file stream.  It also checks for a very specific type of error (`*lfs.CleanedPointerError`), before handling other errors.